src/core/ddsi/include/dds/ddsi/ddsi_plist_generic.h

```diff
@@ -34,6 +34,7 @@ enum pserop {
   Xi, Xix2, Xix3, Xix4, /* int32_t, 1 .. 4 in a row */
   Xu, Xux2, Xux3, Xux4, Xux5, /* uint32_t, 1 .. 5 in a row */
   XD, XDx2, /* duration, 1 .. 2 in a row */
+  Xll,      /* int64_t */
```

Comment:
In my brain, sizeof(long) == 4 and sizeof(long long) == 8. But reality doesn't always bent to the way my brain wants it to work.
I'll change it to ```Xl``` because I have to touch the code anyway.